lib/functional.js

```diff
@@ -303,6 +303,9 @@ module.exports = Functional = {
     });
 
     process.on('uncaughtException', function handleException (error) {
+      if (error.code === 'EPIPE') {
```

Comment:
We should probably log this exit; maybe we keep `console.log(error.stack);` above this `if` statement.

Additionally, I believe we should report a test failure or skipped test when exiting this way.